docs/src/integration-keda.md

```
@@ -0,0 +1,46 @@
+---
+title: Integrations with KEDA
+---
+
+
+
+# Integration with KEDA
+KubeElasti takes care of scaling up and down a service when there is some traffic. KEDA is a good candidate for performing the scaling logic for the service from minReplicas to maxReplicas based on its triggers.
+
+Here we will see how to integrate KubeElasti with KEDA to build a complete scaling solution.
+
+## Prerequisites
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)